### DISCUSSION TITLE
能否出键盘组合鍵?

### DISCUSSION THREAD

**Discussion by xxe753** [ClassicOldSong/moonlight-android#24] [Ideas]
就像的西瓜hub的増强版一样，特殊按键断开连接再恢复串流叓重新开，有点难受。

**Comment by ClassicOldSong** [comment]
不是很明白你说的是什么，键盘组合键跟特殊按键有什么关联吗

**Reply by xxe753** [reply]
我的想法是出一个Ctrl+C或者是Ctrl+v等直组合到一起接显示到屏幕上，就点一下就能完成Ctrl+c的动作了；工作效率就又快了一点，我真的不想用两只手指才能完成Ctrl+C的动作了，或者点好多下

**Comment by ClassicOldSong** [comment]
最新版有了，需要自己加一下 https://github.com/ClassicOldSong/moonlight-android/releases/tag/v12.1-artemis-240929

用法说明： https://github.com/ClassicOldSong/moonlight-android/wiki/Custom-Special-Keys
